db/migrate/20230919075344_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
```

Comment:
### Optional Changes [OPTIONAL]

Hello,

Great job on setting up your migrations. However, there's a small adjustment needed concerning the data type used for the `text` field in the `comments` table.

Currently, the `text` field is using a `string` datatype. While `string` is suitable for shorter text fields, it is generally recommended to use `text` datatype for longer and unbounded strings, such as comments or descriptions, to avoid any limitations in data storage. 

Here’s how you can modify the migration:

```ruby
class CreateComments < ActiveRecord::Migration[7.0]
  def change
    create_table :comments do |t|
      t.text :text  # Changing this line to use :text instead of :string

      t.timestamps
    end
  end
end
```

After modifying the migration, don’t forget to run `rails db:migrate` to apply the changes to your database schema.

Please make this adjustment and continue with your excellent progress!

Best Regards,
Temitope